src/applications/claims-status/components/ClaimTimeline.jsx

```diff
@@ -62,23 +62,7 @@ export default function ClaimTimeline({ currentPhaseBack, id, events, phase }) {
           current={userPhase}
           activity={activityByPhase}
           id={id}
-        >
```

Comment:
Removed as we dont want to show this anymore it now appears on the status tab. Also we never showed this when the toggle was off since the timeline was hidden when a claim was closed.